### PR TITLE
[UVEXGROUP-2203] create plugin

### PR DESCRIPTION
**Pull request by TheBreaken**

### CODE REVIEW

**Review by sd-obermeyer — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+dist: trusty
+sudo: false
+
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+
+matrix:
+  include:
```
Comment (sd-lueckel) [inline .travis.yml:11]: We should directly test php 7.3 and php 7.4 as well

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^5.7",
+        "shopware/shopware": "^5.6",
+        "solutiondrive/sd-test-environment-shopware": "~0.2",
+        "solutiondrive/standalone-build-tools": "^1.0"
+    },
+    "config": {
+        "sort-packages": true,
+        "standalone-build-tools": {
```
Comment (sd-lueckel) [inline composer.json:31]: Here we should use the correct php versions as well

**Review by sd-lueckel — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (TheBreaken) [reply composer.json:31]: What do you mean with the correct php version?

Comment (sd-lueckel) [reply composer.json:31]: See the other comment above in the .travis.yml "We should directly test php 7.3 and php 7.4 as well"

File: .travis.yml (hunk 1 above)
Comment (TheBreaken) [reply .travis.yml:11]: Done https://github.com/netlogix/nlxProductComparison/pull/1/commits/c4aeb2bb669e0429e33fc5d13cc7af0f15954d88

File: composer.json (hunk 2 above)
Comment (TheBreaken) [reply composer.json:31]: Done https://github.com/netlogix/nlxProductComparison/pull/1/commits/c4aeb2bb669e0429e33fc5d13cc7af0f15954d88

**Review by sd-lueckel — APPROVED** [review]

**Pull request merged**